docs/source/actions.rst

```diff
@@ -182,6 +182,51 @@ In the example above, ``to_number`` parameter contains attribute ``secret``
 which value is ``true``. If an attribute is marked as a secret, value of that
 attribute will be masked in the |st2| service logs.
 
+Parameters in actions
```

Comment:
@Kami's comment on my fork:
```
It would be great if we could generate a list of the available base variables for each "execution type" (API call and ChatOps) from code (e.g. something like this - https://github.com/StackStorm/st2/commit/6fb274ccd272c5f4cc31f1ecff0841b89455ca72).

I know this requires some refactoring work since right now context is created inside the method and is not a constant like in the PermissionType case.
```